streams/src/test/java/org/apache/kafka/streams/processor/internals/StateDirectoryTest.java

```diff
@@ -290,15 +307,21 @@ public void shouldNotRemoveNonTaskDirectoriesAndFiles() {
     }
 
     @Test
-    public void shouldListAllTaskDirectories() {
+    public void shouldNotListNonEmptyTaskDirectories() {
```

Comment:
Is the double negative intentional here?

Reply:
Good call, will change.